src/index.html

```diff
@@ -9,5 +9,47 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+    </div>
+    <div class="stars stars--1">
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars--star"></div>
      <div class="stars--star"></div>
      <div class="stars--star"></div>
      <div class="stars--star"></div>
      <div class="stars--star"></div>
    </div>
    
    <div class="stars stars--1">
```

Please add an empty line between multiple sibling blocks of html